### PR TITLE
Scipy2017

### PR DESCRIPTION
**Pull request by binson-b**
Certificate for SciPy2017

### CODE REVIEW

File: certificate/models.py [hunk 1]
```diff
@@ -269,3 +270,18 @@ class Internship16_participant(models.Model):
...
+    ('W','Workshop'),
+    ('T','Organizers')
+    )
+
+class Scipy_2017(models.Model):
+    name = models.CharField(max_length=300)
+    email = models.CharField(max_length=300)
+    paper = models.CharField(max_length=300)
+    purpose = models.CharField(max_length=10, default='S17')
+    attendee_type = models.CharField(max_length=25, choices=(attendee_types)
```
Comment (prathamesh920) [inline certificate/models.py]: Closing bracket missing.
attendee_types should not be in parentheses

File: certificate/views.py [hunk 2]
```diff
@@ -2618,3 +2619,141 @@ def create_python_workshop_certificate(certificate_path, name, qrcode, type, pap
...
+                    filename=%s' % (download_file_name)
+            response.write(pdf.read())
+            _clean_certificate_certificate(certificate_path, file_name)
+            return [response, False]
+        else:
+            error = True
+    except Exception, e:
+        error = True
+    return [None, error]
+    '''pdf = open('{0}{1}.pdf'.format(certificate_path, file_name) , 'r')
```
Comment (prathamesh920) [inline certificate/views.py]: remove this string if not used(Please do not push commented code).

**Review by prathamesh920 — CHANGES_REQUESTED** [review]

File: certificate/models.py (hunk 1 above)
Comment (binson-b) [reply certificate/models.py]: done

File: certificate/views.py (hunk 2 above)
Comment (binson-b) [reply certificate/views.py]: ok sir, Done

**Comment by prathamesh920** [comment]
Merging for now.
Also, organiser certificate will be implemented in next PR

**Review by prathamesh920 — APPROVED** [review]

**Pull request merged**